Sources/Caching/DeviceCache.swift

```diff
@@ -368,6 +384,7 @@ class DeviceCache {
             case let .offerings(userID): return "\(Self.base)offerings.\(userID)"
             case let .legacySubscriberAttributes(userID): return "\(Self.legacySubscriberAttributesBase)\(userID)"
             case let .attributionDataDefaults(userID): return "\(Self.base)attribution.\(userID)"
+            case let .syncedSK2TransactionIDs(userID): return "\(Self.base)syncedSK2TransactionIDs.\(userID)"
```

Comment:
I'd add something regarding observer mode to this key so its purpose is clearer (e.g. `latestSyncedSK2ObserverModeTransactionID`)